lib/node_modules/@stdlib/number/float16/base/signbit/manifest.json

```
@@ -0,0 +1,81 @@
+{
+  "options": {
+    "task": "build",
+    "wasm": false
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/number/float16/base/to-word",
+        "@stdlib/number/float32/base/to-float16",
```

```suggestion
        "@stdlib/number/float64/base/to-float16",
```
You are using `stdlib_base_float64_to_float16` in addon.c